modules/network-observability-operator-install.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-operator-installation_{context}"]
+
+= Installing the Network Observability operator
+You can install the Network Observability operator using the {product-title} web console Operator Hub.
+ 
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+//In the Operator Hub on the test console Julian provided perms for, its NetObserv Operator, with a capital O. 
+.. Choose  *Network Observability operator* from the list of available Operators in the *OperatorHub*, and click *Install*.
```

Comment:
```suggestion
.. Choose  *Network Observability Operator* from the list of available Operators in the *OperatorHub*, and click *Install*.
```

```suggestion
```
Yes, the upstream operator (currently the only one available) is `NetObserv Operator`
The downstream one will be `Network Observability Operator`

Reply:
ACK, thanks for clarifying O vs o.

Reply:
According to legal, the approved name is network observability.

cc @skrthomas

Reply:
@davegord so no "operator" after? We've been having some discussion about this legal/marketing language guidance as well. In docs, we usually title operators as such, with a capital O.

Reply:
@bergerhoffer tagging you here re: one of the operator capitalization questions.